test/system/120-load.bats

```diff
@@ -38,9 +47,26 @@ verify_iid_and_name() {
     fi
 
     # Make sure we can reload it
-    # FIXME: when/if 7337 gets fixed, add a random tag instead of rmi'ing
-    # FIXME: when/if 7371 gets fixed, use verify_iid_and_name()
+    # load oci-archive and use the name to override it
     run_podman rmi $iid
+
+    run_podman load -i $archive foo
+    verify_name localhost/foo:latest
+    run_podman rmi localhost/foo:latest
+
+    run_podman load -i $archive foo:test
+    verify_name localhost/foo:test
+    run_podman rmi localhost/foo:test
+
+    run_podman load -i $archive test.com/foo
+    verify_name test.com/foo:latest
+    run_podman rmi test.com/foo:latest
+
+    run_podman load -i $archive test.com/foo:test
+    verify_name test.com/foo:test
+    run_podman rmi test.com/foo:test
```

Comment:
This is a nice set of tests, but they're actually unrelated to the purpose of this particular test block ("save to pipe"). They would belong better in a separate `@test`. I would also like to see an explicit test for CAPS in the archive name. So maybe something like:
```bats
@test "podman save/load: oci-archive" {
    # FIXME: remove this if #7371 gets fixed
    safe_archive=$PODMAN_TMPDIR/restoreme.tar
    run_podman save -o $safe_archive $IMAGE

    archive=$PODMAN_TMPDIR/MYIMAGE-$(random_string 8).tar

    run_podman save --format oci-archive -o $archive $IMAGE
    run_podman rmi $IMAGE

    function _load_and_verify() {
        run_podman load -i $archive "$1"
        run_podman images -a --format '{{.Repository}}:{{.Tag}}'
        is "$output" "$2" "podman load, with tag $1"
        run_podman rmi "$2"
    }

    _load_and_verify "foo" "localhost/foo:latest"
    _load_and_verify "foo:test" "localhost/foo:test"
    _load_and_verify "test.com/foo" "test.com/foo:latest"
    _load_and_verify "test.com/foo:test" "test.com/foo:test"

    # Restore original test image with real IID (FIXME: #7371)
    run_podman load -i $safe_archive
}

Reply:
I got it